.probo.yaml

```diff
@@ -113,13 +113,21 @@ steps:
       - source $ASSET_DIR/test-credentials.sh
       - export SIMPLETEST_BASE_URL="http://localhost"
       - export SIMPLETEST_DB="mysql://${TEST_DATABASE_USER}:${TEST_DATABASE_PASS}@localhost/${TEST_DATABASE_NAME}"
-      - export MINK_DRIVER_ARGS_WEBDRIVER='["chrome", {"browserName":"chrome","chromeOptions":{"args":["--disable-gpu","--disable-dev-shm-usage","--headless","--no-sandbox"]}}, "http://localhost:9515"]'
+      - export MINK_DRIVER_ARGS_WEBDRIVER='["chrome", {"browserName":"chrome","goog:chromeOptions":{"args":["--disable-gpu","--disable-dev-shm-usage","--headless","--no-sandbox"]}}, "http://localhost:9515"]'
       - daemonize /usr/local/bin/chromedriver --log-path=/tmp/chromedriver.log --verbose --whitelisted-ips="127.0.0.1"
       - cd /var/www/html/profiles/custom/az_quickstart
       - export RUN_TESTS_CONCURRENCY=10
       - mysqladmin create ${TEST_DATABASE_NAME}
       - sudo -u www-data -E find $SRC_DIR/az-quickstart-scaffolding/web/profiles/custom/az_quickstart/ -name "*Test.php" | sudo -u www-data -E $SRC_DIR/az-quickstart-scaffolding/vendor/bin/fastest -p 10 -vvv "$SRC_DIR/az-quickstart-scaffolding/vendor/bin/phpunit {} --colors="always" --verbose --do-not-cache-result;"
-  - name: Log in to terminus
+  - name: Install Terminus
+    plugin: Script
+    script:
+      - mkdir -p ~/terminus && cd ~/terminus
+      - curl -L https://github.com/pantheon-systems/terminus/releases/download/3.4.0/terminus.phar --output terminus
```

Comment:
this pins to 3.4.0, hope that is OK

Reply:
Ohh good point....  I think we just needed a newer (PHP 8.3 compatible version) than what is available by default on Probo.  I'll see if we can get just use the latest version instead.

Reply:
Actually, we already have a self:update step a little later.